src/components/dialogs/AddStaffNotationDialog.vue

```diff
@@ -235,4 +255,10 @@ export default class AddStaffNotationDialog extends Mixins(DateMixin) {
   color: $gray7;
   font-size: $px-16;
 }
+.dialog-text {
+  padding-bottom: 10px;
```

Comment:
Most of our HTML uses top whitespace (on the next component) instead of bottom whitespace.

It's better to stick with all-top or all-bottom whitespace for layout purposes to handle conditional blocks (and also, as a de facto style, easier to update in the future).

Reply:
switched to 'padding-top' for component below (notation-text)